src/conditionals/programs_with_decision_points/hard/q8/PointsScoring.java

```diff
@@ -0,0 +1,44 @@
+package conditionals.programs_with_decision_points.hard.q8;
+
+/*
+ * Write a Java Program that asks a player for their score, and prints out the prize that a player wins in a game
+ * depending on their score. If the player scores 5000 points or above, they win a gold prize. If the player scores
+ * between 2500 - 4999 points, they win a silver prize. If the player scores between 1000 - 2499 points, they win a
+ * bronze prize. If they score below 1000 points, they don't win any prize. Make sure to print out an error message if 
+ * the player enters an invalid score.
```

Comment:
I would change the `Make sure to print an error message ... ` sentence because the question does not specify what is a valid score. If we want to make sure the score is >=  0, just change it to `Make sure to check if the score is greater than or equal to 0, and print out an error message if it is not.`